readout.sh

```diff
@@ -1,6 +1,6 @@
 package: Readout
 version: "%(tag_basename)s"
-tag: v2.0.2
+tag: v2.1,0
```

Comment:
There's a comma instead of a dot here ;)

Reply:
indeed, thanks !